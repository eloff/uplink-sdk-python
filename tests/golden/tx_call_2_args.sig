r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 105586503817136353979400346932986385078768783025016749267900346462882178223361
