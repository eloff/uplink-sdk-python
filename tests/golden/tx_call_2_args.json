{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                "args": [
                    {
                        "contents": 1, 
                        "tag": "VInt"
                    }, 
                    {
                        "contents": 1, 
                        "tag": "VInt"
                    }
                ], 
                "method": "get"
            }, 
            "tag": "Call"
        }, 
        "tag": "TxContract"
    }, 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE4xMDU1ODY1MDM4MTcxMzYzNTM5Nzk0MDAzNDY5MzI5ODYzODUwNzg3Njg3ODMwMjUwMTY3NDkyNjc5MDAzNDY0NjI4ODIxNzgyMjMzNjE="
}